### ISSUE TITLE
[Visual requirement – Make Code – Language] Luminosity ratio is less than 3:1 for focus indicator and border line for 'close' controls present on  pop-up. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3760]
User Experience:
Visual impaired user will find difficulty to see the focus indicator over the control or element and border line for any control if that focus indicator and border line have insufficient color contrast ratio as user won't be able to proceed further.

Note: User credentials should NOT be included in the bug.

Repro Steps:
Launch application URL :
Home page will open.
Navigate to "Import" button and select it.
Navigate through the controls appearing under "Import" dialog.
Navigate to "Your GitHub Repo" option and  Select it.
Navigate and select "Sign In" control under Sign in with GitHub and navigate to all the controls of dialogue "Clone or create your own GitHub repo
Navigate and select "Create New" control under "Clone or create your own GitHub repo" and navigate to all the controls of dialogue "Create GitHub Repository
Verify the issue.

Actual Result:
Luminosity ratio is less than 3:1 for focus indicator and border line of 'close' controls present on  pop-up and due to this, visual impaired user is unable to see the focus indicator and border line.

Expected Result:
Luminosity ratio should be more than and equal to 3:1 for focus indicator and border line of 'close' controls present pop-up, so that visual impaired user could easily see the border of focus.

Note:
Same issue exists on Firefox & Chrome.
Same issue exists throughout the application.

WCAG Reference:
[WCAG 1.4.11 - Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG1.4.11; #Win10-Edge(Chromium)

![WCAG1 4 11_Luminosity ratio is less](https://user-images.githubusercontent.com/72871089/100215985-be755780-2f37-11eb-96d5-9cb9c6709f00.png)

**Comment by jwunderl** [comment]
@livcheerful fixed this in https://github.com/microsoft/pxt/pull/8058
